GRADLE_STRUCTURE.md

```diff
@@ -0,0 +1,16 @@
+# Gradle Structure
+
+`package/example/android/settings.gradle` imports the flutter tooling and includes the `app` directory and it configures GoogleCloudPlatform/artifact-registry-maven-tools for use in ci.
```

Comment:
Very nitpicky but I think this sentence would sound better as:
`package/example/android/settings.gradle` imports the flutter tooling, includes the `app` directory, and configures GoogleCloudPlatform/artifact-registry-maven-tools for use in ci.

Reply:
Done!